tests/unit/test_pynblint.py

```diff
@@ -1,6 +1,154 @@
 from pynblint.notebook import Notebook
 from pathlib import Path
+import pytest
 
 if __name__ == '__main__':
-    nb = Notebook(Path('../fixtures/my-attempt-at-analytics-vidhya-job-a-thon.ipynb'))
-    print(nb.get_pynblint_results())
+    pytest.main()
+
+
+@pytest.fixture(scope="module")
+def notebooks():
+    nb1 = Notebook(Path("../fixtures") / Path("EmptyNotebook.ipynb"))
+    nb2 = Notebook(Path("../fixtures") / Path("FullNotebook.ipynb"))
+    nb3 = Notebook(Path("../fixtures") / Path("JustMarkdown.ipynb"))
+    nb4 = Notebook(Path("../fixtures") / Path("DefectedNotebook.ipynb"))
+    return {
+        "EmptyNotebook.ipynb": nb1,
+        "FullNotebook.ipynb": nb2,
+        "JustMarkdown.ipynb": nb3,
```

Comment:
This notebook contains just one _empty_ Markdown cell. Do we actually need it?

Reply:
having a notebook with just markdown is the only way to have a notebook without code cells

Reply:
Ok, but is there any reason for having an empty Markdown cell as the only Markdown cell in the notebook? It seems quite strange. Anyway, if you need it like so, that's OK.